codesign/bin/codesign.dart

```diff
@@ -2,8 +2,99 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-import 'package:codesign/codesign.dart' as codesign;
+import 'dart:io';
 
-void main(List<String> arguments) {
-  print('Hello world: ${codesign.calculate()}!');
+import 'package:args/args.dart';
+import 'package:codesign/codesign.dart';
+import 'package:platform/platform.dart';
+
+/// Definitions of variables are included in help texts below.
```

Comment:
Are these all cli flags? If so, we should add `Flag` at the end of each variable's name to make it clearer